X/xrootdgo/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+using BinaryBuilder
+
+name = "xrootdgo"
+version = v"0.30.1"
+
+sources = [
+    GitSource("https://github.com/go-hep/hep/",
+              "76e7c8652ee045064931dec627aedd33b4457fee"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+install_license ${WORKSPACE}/srcdir/hep/LICENSE
+cd $WORKSPACE/srcdir/hep/
+
+if [[ "${target}" == *-darwin* ]]; then
+    # Create official Apple-blessed `xcrun`, needed by the CGO linker
+    cat > /usr/bin/xcrun << EOF
+#!/bin/sh
+exec "\${@}"
+EOF
+    chmod +x /usr/bin/xcrun
+fi
+
```

Comment:
I don't think this is needed anymore, we do it directly in BBB: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/b688ad040ee699c6f79fcf1056fe35f28a47ef04/src/Runner.jl#L837-L845